### DISCUSSION TITLE
Permission to add

### DISCUSSION THREAD

**Discussion by dvkam** [kevinelliott/awesome-tezos#3] [Ideas]
Hi, I added some new things to the list and would push in a new branch for your review, can I get the access rights?

**Comment by kevinelliott** [comment]
Hi @Reudlak ... what you do is Fork this repository, make the changes in a branch on your fork, and then submit a Pull Request from your fork/branch to here. Looking forward to your contribution!

**Reply by dvkam** [reply]
Ok thanks! Added some as start and if thats fine this way I will contribute more
